docs/usage/customization-guide.md

```diff
@@ -594,6 +594,45 @@ details.
 > labels specified in the `labels` field will override anything
 > originating from `labelsTemplate`.
 
+#### Node Annotations
+
+The `.annotations` field is a list of features to be advertised as annotations.
+
+Take this rule as a referential example:
+
+```yaml
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: feature-annotations-example
+spec:
+  rules:
+    - name: "annotation-example"
+      annotations:
+        feature.node.kubernetes.io/defaul-ns-annotation: "foo"defaul-ns-annotation: "foo"
```

Comment:
This is not legit. I wasn't suggesting to change this example NFR but correct the "result" yaml below (as that's the one that's incorrect).

Also, please add a note about the namespace/prefix restrictions (https://github.com/kubernetes-sigs/node-feature-discovery/pull/1417#discussion_r1370594349)

Reply:
ah my bad, and also... weird thing there

Reply:
fixing....